core/kernel/console.c

```diff
@@ -20,6 +21,11 @@ static struct serial_chip *serial_console __nex_bss;
 
 void console_init(void)
 {
+	if (IS_ENABLED(CFG_SEMIHOSTING_CONSOLE)) {
+		semihosting_console_init(CFG_SEMIHOSTING_CONSOLE_FILE);
+		return;
```

Comment:
nitpicking: prefer:

	if (IS_ENABLED(CFG_SEMIHOSTING_CONSOLE))
		semihosting_console_init(CFG_SEMIHOSTING_CONSOLE_FILE);
	else
		plat_console_init();

Reply:
Apply in https://github.com/OP-TEE/optee_os/pull/6700/commits/f542f6c9e8bb12571ee922bbb715b698c084ee4c#diff-49b28b57c451219e3e5f3f7b51994f0d424338fdb7b0134f2217bd44dfdd1081